### DISCUSSION TITLE
if I make all the code for a weapon would it be added?

### DISCUSSION THREAD

**Discussion by Crimson-Haze** [HasangerGames/suroi#431] [Q&A] [answered]
also if I could join the dev team that would be amazing, I love this game and want to help it grow so if I can help in any way I will do so.

**Comment by Hashimk2** [comment, marked as answer]
the maintainers have to agree and then you make a pull request
then wait and see if hasanger approves

**Reply by Crimson-Haze** [reply]
who are the maintainers?

**Reply by Crimson-Haze** [reply]
also I did it now what?

**Reply by Hashimk2** [reply]
> who are the maintainers?
Go to the first page of github suroi, and it says the contributors, there are 42 contributors

**Reply by Crimson-Haze** [reply]
now what?
